docs/ibm-cloud-rules.md

```diff
@@ -126,6 +127,13 @@ Instead, use a ref to a named schema.
 <td>oas3</td>
 </tr>
 <tr>
+<td><a href="#ibm-avoid-multiple-types">ibm-avoid-multiple-types</a></td>
+<td>warn</td>
+<td>Multiple types within a schema <code>type</code> field should be avoided due to potential impact on SDK code generation.
```

Comment:
I don't think the purpose of this rule is for code generation only - it will be discouraged in general API use as well due to the ambiguity it creates.

Reply:
fixed